server/src/services/customers.ts

```diff
@@ -14,10 +14,25 @@
  * limitations under the License.
  */
 
-import {CustomerResponse} from '../interfaces';
+import {DEFAULT_CUSTOMER_PAYLOAD} from '../constants/default-payload';
+import {CustomerResponse, CustomerPayload} from '../interfaces';
 import {customerStorage} from '../storage';
 
 const getCustomer = async (customerId: number): Promise<CustomerResponse> =>
   await customerStorage.getCustomer(customerId);
 
-export default {getCustomer};
+/**
+ * Creates a customer if the customer is not already registered, and returns customer information.
```

Comment:
Can we add more doc about what happens if the customer is already registered? Like does it throw exception?